### ISSUE TITLE
训练好的模型 Rank@1：0.9269 | Rank@5：0.9774 | mAP：0.7963

### ISSUE THREAD

**Issue by vdjango** [layumi/Person_reID_baseline_pytorch#186]
| Rank@1  | Rank@5| mAP | Reference |
| :--| :-- | :-- | :-- |
| 0.9269 | 0.9774 | 0.7963 | python3.6 train.py --batchsize 32 --lr 0.01 --PCB --erasing_p 0.5 --name PCB64lr01 --data_dir ../../data/pytorch/ |

训练时，没有变化了，基本浮动在这个范围。

loss: 4.8
acc: 0.91

不知道什么情况，为啥怎么高，loss

链接:https://pan.baidu.com/s/19K0XnifYLCgWr7wxGJuGNg  密码:nocv

**Comment by layumi** [comment]
你好，这个结果是你测的么？感觉差不多啊。没有PCB的情况map是78多。你这个测试79.6应该差不多。

loss的话，也正常PCB的话，是几个part loss的sum，你可以去看看代码。我忘记是不是这么大了，，

**Comment by vdjango** [comment]
是我训练的，跑了2天，第一天降到loss4.&nbsp;9，第一天晚上到第二天就没有变化了，基本保持不变


---原始邮件---
发件人: &quot;Zhedong Zheng&quot;<notifications@github.com&gt;
发送时间: 2020年1月19日(周日) 下午5:53
收件人: &quot;layumi/Person_reID_baseline_pytorch&quot;<Person_reID_baseline_pytorch@noreply.github.com&gt;;
抄送: &quot;小敏子&quot;<vdjango@vip.qq.com&gt;;&quot;Author&quot;<author@noreply.github.com&gt;;
主题: Re: [layumi/Person_reID_baseline_pytorch] 训练好的模型 Rank@1：0.9269 | Rank@5：0.9774 | mAP：0.7963 (#186)



你好，这个结果是你测的么？感觉差不多啊。没有PCB的情况map是78多。你这个测试79.6应该差不多。
 
loss的话，也正常PCB的话，是几个part loss的sum，你可以去看看代码。我忘记是不是这么大了，，
 
&mdash;

**Comment by vdjango** [comment]
part是6


---原始邮件---
发件人: &quot;Zhedong Zheng&quot;<notifications@github.com&gt;
发送时间: 2020年1月19日(周日) 下午5:53
收件人: &quot;layumi/Person_reID_baseline_pytorch&quot;<Person_reID_baseline_pytorch@noreply.github.com&gt;;
抄送: &quot;小敏子&quot;<vdjango@vip.qq.com&gt;;&quot;Author&quot;<author@noreply.github.com&gt;;
主题: Re: [layumi/Person_reID_baseline_pytorch] 训练好的模型 Rank@1：0.9269 | Rank@5：0.9774 | mAP：0.7963 (#186)



你好，这个结果是你测的么？感觉差不多啊。没有PCB的情况map是78多。你这个测试79.6应该差不多。
 
loss的话，也正常PCB的话，是几个part loss的sum，你可以去看看代码。我忘记是不是这么大了，，
 
&mdash;

**Comment by vdjango** [comment]
part是6


---原始邮件---
发件人: &quot;小敏子&quot;<notifications@github.com&gt;
发送时间: 2020年1月20日(周一) 上午8:26
收件人: &quot;layumi/Person_reID_baseline_pytorch&quot;<Person_reID_baseline_pytorch@noreply.github.com&gt;;
抄送: &quot;小敏子&quot;<vdjango@vip.qq.com&gt;;&quot;Your activity&quot;<your_activity@noreply.github.com&gt;;
主题: Re: [layumi/Person_reID_baseline_pytorch] 训练好的模型 Rank@1：0.9269 | Rank@5：0.9774 | mAP：0.7963 (#186)


是我训练的，跑了2天，第一天降到loss4.&amp;nbsp;9，第一天晚上到第&#x4E8C;天就没有变化了，基本保持不变 
 
 
 ---原始邮&#x4EF6;--- 
 发&#x4EF6;人: &amp;quot;Zhedong Zheng&amp;quot;<notifications@github.com&amp;gt; 
 发送时间: 2020年1月19日(周日) 下午5:53 
 收&#x4EF6;人: &amp;quot;layumi/Person_reID_baseline_pytorch&amp;quot;<Person_reID_baseline_pytorch@noreply.github.com&amp;gt;; 
 抄送: &amp;quot;小敏子&amp;quot;<vdjango@vip.qq.com&amp;gt;;&amp;quot;Author&amp;quot;<author@noreply.github.com&amp;gt;; 
 主题: Re: [layumi/Person_reID_baseline_pytorch] 训练好的模型 Rank@1：0.9269 | Rank@5：0.9774 | mAP：0.7963 (#186) 
 
 
 
 你好，这个结果是你测的么？感觉差不多啊。没有PCB的情况map是78多。你这个测试79.6应该差不多。 
   
 loss的话，也正常PCB的话，是几个part loss的sum，你可以去看看代码。我忘记是不是这么大了，， 
   
 &amp;mdash; 
 You are receiving this because you authored the thread. 
 Reply to this email directly, view it on GitHub, or unsubscribe. 
&mdash;

**Comment by layumi** [comment]
6个part loss大是可能的，你去看看每一个part的loss，可能最上面（第一个part）和最下面（第六个part）两个 loss比较大。

**Comment by vdjango** [comment]
好，


---原始邮件---
发件人: &quot;Zhedong Zheng&quot;<notifications@github.com&gt;
发送时间: 2020年1月20日(周一) 上午8:47
收件人: &quot;layumi/Person_reID_baseline_pytorch&quot;<Person_reID_baseline_pytorch@noreply.github.com&gt;;
抄送: &quot;小敏子&quot;<vdjango@vip.qq.com&gt;;&quot;Author&quot;<author@noreply.github.com&gt;;
主题: Re: [layumi/Person_reID_baseline_pytorch] 训练好的模型 Rank@1：0.9269 | Rank@5：0.9774 | mAP：0.7963 (#186)



6个part loss大是可能的，你去看看每一个part的loss，可能最上面（第一个part）和最下面（第六个part）两个 loss比较大。
 
&mdash;

**Comment by vdjango** [comment]
我已经将这个reid应用到视频上了，输入一个视频，分析出每一个人的reid。但是效率太低了。一个20分钟视频，30帧。nvidia&nbsp;rtx2080&nbsp;需要跑10个小时。

跑的太慢了，如果想要达到实时效果。有什么方法，降帧我知道，我想先最大的优化，最后在降帧处理

一帧19人，需要3秒时间去处理。通过reid识别人物，做到每一帧与后一帧的每个人的对比，得到人物的ID。
这样做速度太慢，因为要遍历每张图片中的N个人物去跑模型。一帧19人，需要跑19次模型（一帧）。
yolo&nbsp;则是一帧直接输入模型&nbsp;然后输出结果&nbsp;快太多了。

不知道有没有相关实现


---原始邮件---
发件人: &quot;Zhedong Zheng&quot;<notifications@github.com&gt;
发送时间: 2020年1月20日(周一) 上午8:47
收件人: &quot;layumi/Person_reID_baseline_pytorch&quot;<Person_reID_baseline_pytorch@noreply.github.com&gt;;
抄送: &quot;小敏子&quot;<vdjango@vip.qq.com&gt;;&quot;Author&quot;<author@noreply.github.com&gt;;
主题: Re: [layumi/Person_reID_baseline_pytorch] 训练好的模型 Rank@1：0.9269 | Rank@5：0.9774 | mAP：0.7963 (#186)



6个part loss大是可能的，你去看看每一个part的loss，可能最上面（第一个part）和最下面（第六个part）两个 loss比较大。
 
&mdash;

**Comment by layumi** [comment]
@vdjango 
你可以尝试把模型转换到float16 （可以利用上20系的tensorcore）
同时可以尝试TensorRT
